sailthru/requirements/base.txt

```diff
@@ -1,2 +1,3 @@
+nose==1.3.7
```

Comment:
Use a nested file. If I am running the Sailthru script, there is no reason I should need to install pandas, needed by the audit script. The same goes for needing to install Sailthru when running the audit script. Similarly, when running this in production, we should not bother installing test requirements. This particular requirement belongs in `sailthru/requirements/test.txt`.

Also, please file a ticket to enable Travis on this repo. There is no point in having tests if we don't run them.